res/values/strings.xml

```diff
@@ -301,27 +301,27 @@
 
     <!-- welcome and login -->
     <string name="welcome_intro1_headline">Delta Chat</string>
-    <string name="welcome_intro1_message">The messenger with the broadest audience in the world. Free and independent.</string>
+    <string name="welcome_intro1_message">The messenger with the broadest audience in the world. Libre and independent.</string>
     <string name="welcome_intro2_headline">Independent</string>
     <string name="welcome_intro2_message">There is no Delta Chat server. The app only uses your mail server.</string>
     <string name="welcome_intro3_headline">Fast</string>
     <string name="welcome_intro3_message">Push messages in seconds.\nRapid interface.</string>
     <string name="welcome_intro4_headline">Powerful</string>
     <string name="welcome_intro4_message">Unlimited chats, images, videos, audio messages and more. Multi-client capable.</string>
-    <!-- Translators: "free" as in freedom here, not "free" as in "free beer" -->
-    <string name="welcome_intro5_headline">Free</string>
-    <!-- Translators: "free" as in freedom here, not "free" as in "free beer" -->
-    <string name="welcome_intro5_message">Delta Chat is free forever. Open-source. No ads. No subscription. No vendor lock-in.</string>
+    <!-- Translators: libre sofware as in use, see, change and share freely -->
+    <string name="welcome_intro5_headline">Libre</string>
+    <!-- Translators: https://en.wikipedia.org/wiki/Copyleft meaning it has to be shared with everyone, and https://en.wikipedia.org/wiki/Free_software -->
+    <string name="welcome_intro5_message">Delta Chat is copylefted libre software. It has no ads, subscription or vendor lock-in.</string>
```

Comment:
same here. also the wikipedia link say mpl is not copyleft. but this should be discussed elsewhere.

Reply:
Weak copyleft is still copyleft.